### DISCUSSION TITLE
SOC II Certification

### DISCUSSION THREAD

**Discussion by danmalone89** [supabase/supabase#3017] [Questions] [answered]
Is there an expected date when Supabase will be in SOC II compliance?

**Comment by thebengeu** [comment, marked as answer]
We are working on being compliant and are targeting Q1 2022 to be SOC 2 Type 1 compliant.

**Reply by J0** [reply]
Supabase is currently SOC2 Type 1 compliant and we will be pursuing a type 2 soon.

More details here: https://supabase.com/blog/supabase-soc2
